cli/src/factory_ops/boxing/init

```diff
@@ -95,6 +95,17 @@ fi
 # Prepare shutdown
 ensure_healthy_cluster
 
+echo "INFO: Resolving service user credentials" | tee -a ${LOG_FILE}
+# FIXME these values might be encrypted
+service_user_name=$(salt-call pillar.get system:service_user:name --output=newline_values_only)
```

Comment:
Please use --local option to salt-call commands.

Reply:
I copied from [here](https://github.com/Seagate/cortx-prvsnr/blob/e6d15babe156505f810b912ccd0918112cc42788/cli/src/factory_ops/boxing/prov_tasks#L76). What is the difference?